frontend/src/services/auth/types.ts

```diff
@@ -1,7 +1,7 @@
 export interface AuthService {
   currentUser(): Promise<User | null>;
   signUp(info: SignUpInfo): Promise<void>;
-  signIn(credentials: LoginCredentials): Promise<User>;
+  Login(credentials: LoginCredentials): Promise<User>;
```

Comment:
Can we make this to camelCase (ie. `login`) to keep it consistent with other method names?

Reply:
You might also have to update where the `Login` method is being called to `login`